src/Symfony/Component/Form/Tests/CompoundFormTest.php

```diff
@@ -630,21 +630,21 @@ public function testSubmitPostOrPutRequest($method)
 
         $form->handleRequest($request);
 
-        $file = new UploadedFile($path, 'upload.png', 'image/png', null, UPLOAD_ERR_OK);
+        $file = new UploadedFile($path, 'upload.png', 'image/png', UPLOAD_ERR_OK);
 
         $this->assertEquals('Bernhard', $form['name']->getData());
         $this->assertEquals($file, $form['image']->getData());
 
         unlink($path);
     }
 
-    /**
-     * @dataProvider requestMethodProvider
+    /** @dataProvider requestMethodProvider
      */
     public function testSubmitPostOrPutRequestWithEmptyRootFormName($method)
     {
         $path = tempnam(sys_get_temp_dir(), 'sf2');
         touch($path);
```

Comment:
we can now drop the `touch()` call everywhere if I don't miss anything

Reply:
I needed to add some in order to make some test work, I don't now if we can remove them ill check